cvmfs/publish/repository_transaction.cc

```diff
@@ -115,8 +116,16 @@ void Publisher::TransactionImpl() {
              settings_.transaction().template_from().c_str(),
              settings_.transaction().template_to().c_str());
     ConstructSyncManagers();
-    catalog_mgr_->CloneTree(settings_.transaction().template_from(),
-                            settings_.transaction().template_to());
+
+    try {
+      catalog_mgr_->CloneTree(settings_.transaction().template_from(),
+                              settings_.transaction().template_to());
+    } catch (const ECvmfsException &e) {
+      LogCvmfs(kLogCvmfs, kLogStdout, "%s", e.what());
```

Comment:
Rather than logging the error here, let's append it to the `EPublish` exception.